client/components/questionnaire1.js

```diff
@@ -26,25 +43,44 @@ class Questionnaire1 extends Component {
     const questionnairePage1 = [
       {
         questionNumber: 1,
-        name: 'income',
-        title: 'Please enter your earnings per month',
-        value: this.state.income,
-        isRequired: true
+        name: 'Food',
+        title:
+          'How much do you typically spend each month on food? (including groceries, drinks, and dining out,)',
+        value: this.state.food.overallMonthly,
+        isRequired: true,
```

Comment:
Consider adding the category database attribute name as a different key on the questionnaire array? 
{
questionNumber: 1,
...,
category: 'food'
}

Reply:
This might not be necessary since I have the handleChange() function performing that logic. We could refactor it later, but for right now it does work with multi word categories.